### PR TITLE
Restyle Blog Page

### PR DESCRIPTION
**Pull request by jhancock532**
#### [Ticket on Monday](https://torchbox.monday.com/boards/2604174158/pulses/2605689247)

Adds a custom design for the blog page, verified with Ben Tuckwell. Updates the design of the streamfield block pictures.

### Screenshots
<details><summary>Expand to view more</summary>

8080-jhancock532-bakerydemo-u3xd46cm0gj ws-eu44 gitpod io_blog_icelandic-baking_ (1)

8080-jhancock532-bakerydemo-u3xd46cm0gj ws-eu44 gitpod io_blog_icelandic-baking_(iPhone SE) (1)

<a href="#">Back to top</a>
</details>

### Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Consider updating documentation. If you don't, tell us why.
- [x] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

File: bakerydemo/static/css/main.css [hunk 1]
```diff
@@ -127,7 +127,8 @@ p {
   }
 }
 
-ol, ul {
+ol,
+ul {
```
Comment (jhancock532) [inline bakerydemo/static/css/main.css:145]: A few minor changes caused by auto-format on save.

File: bakerydemo/static/css/main.css [hunk 2]
```diff
@@ -363,9 +370,7 @@ blockquote footer::before {
   display: none;
 }
 
-blockquote footer p {
-  
-}
+blockquote footer p {}
```
Comment (jhancock532) [inline bakerydemo/static/css/main.css:387]: Deleted in another MR

File: bakerydemo/templates/blog/blog_page.html [hunk 3]
```diff
@@ -3,42 +3,34 @@
...
-                            {{ page.date_published }}
-                        </div>
-                    {% endif %}
+<div class="container">
+    <div class="row">
+        <div class="col-md-8">
+            <div class="blog__meta">
+                {% if page.authors %}
+                <div class="blog__avatars">
+                    {% for author in page.authors|slice:":1" %}
```
Comment (jhancock532) [inline bakerydemo/templates/blog/blog_page.html:15]: Added limit to one author after feedback from Ben Tuckwell

File: bakerydemo/templates/blog/blog_page.html [hunk 4]
```diff
@@ -3,42 +3,34 @@
...
-                    Tagged with:<br />
-                    {% for tag in page.get_tags  %}
-                        <a href="{{ tag.url }}" class="btn btn-sm">{{ tag }}</a>
-                    {% endfor %}
-                {% endif %}
+            {% if page.get_tags %}
+            <p class="blog__tag-introduction">Find more blog posts with similar tags</p>
+            <div class="blog-tags blog-tags--condensed">
+                {% for tag in page.get_tags %}
+                <a href="{{ tag.url }}" class="blog-tags__pill">{{ tag }}</a>
```
Comment (chris-lawton) [inline bakerydemo/templates/blog/blog_page.html:30]: suggestion: add context for a screenreader here to say what will happen on click

**Review by chris-lawton — APPROVED** [review]
Nice work James!

**Pull request merged**